src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -159,8 +159,31 @@
 
 (def channel-list-component (memoize channel-list-component-fn))
 
+(defn join-community [{:keys [joined can-join? requested-to-join-at community-color] :as community}]
+  (let [node-offline? (and can-join? (not joined) (pos? requested-to-join-at))]
+    [:<>
+     (when-not joined
+       [quo/button
+        {:on-press  #(rf/dispatch [:bottom-sheet/show-sheet
+                                   {:content (constantly [requests.actions/request-to-join community])
```

Comment:
why constantly? its hard to read here

Reply:
This part was extracted as-is from a bigger component)

Reply:
could you change it please?

Reply:
sure!

Reply:
Humm, let's not generalize that `constantly` is hard to read. This is just personal preference. I find it quite clear in many contexts, and it fits like a glove in this piece of code. But of course I'm okay with switching to just `(fn [] [requests.actions/request-to-join community])`

Reply:
i mean we've never used `constantly` so im not used to it, so we need to use the same pattern for all such cases in the code, `constantly` is good in java i guess , to safely have a function with any number of arguments if you don't use them, in javascript we don't case so `constantly` is useless